server/models/card/cardQuery.ts

```diff
@@ -64,6 +64,11 @@ const cards: ArgumentField<CardsArgs> = {
           });
         break;
       default:
+        payload = payload.sort((a, b) => {
```

Comment:
What about putting this logic [here](https://github.com/aliemteam/aliemcards/blob/eeb5ed39f99c69d70289decc19faf00cebb12770/server/utils/normalize.ts#L38) instead? That way it'll only have to run once.